src/cli/VariablesCLIController.ts

```diff
@@ -43,6 +43,11 @@ export class VariablesCLIController extends BaseCLIController {
   }
   
   public async getAllVariables() {
+    // when the current organization is changed in the Home View, current project is undefined until a new one is selected
+    const project_id = StateManager.getFolderState(this.folder.name, KEYS.PROJECT_ID)
+    if (!StateManager.getFolderState(this.folder.name, KEYS.PROJECT_ID)) {
```

Comment:
was this line intended to use the `project_id` var from the line above?

Reply:
nope. Leftover from debugging. Removed